services/libs/integrations/src/integrations/activityTypes.ts

```diff
@@ -478,6 +480,18 @@ export const DEFAULT_ACTIVITY_TYPE_SETTINGS: DefaultActivityTypes = {
       isContribution: true,
     },
   },
+  [PlatformType.YOUTUBE]: {
```

Comment:
The integration is defined here but for some reason the short message showing on the clients is still "conducted an activity" not sure why, should I define this somewhere else ?